src/arrayMethodSort.js

```diff
@@ -5,7 +5,35 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
you can just add a default value for compare function 
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => String(a) > String(b)) {
```